scripts/irods/test/resource_suite.py

```diff
@@ -519,6 +519,7 @@ def test_local_iput_physicalpath_no_permission(self):
         if os.path.exists(datafilename):
             os.unlink(datafilename)
 
+    @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, "Skip for Topology Testing")
```

Comment:
@alanking why would this need to be skipped?   we're not looking in the vault...